components/release/release-service-configurator-role.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
I missed before but testing the kustomize build now, I see that you also need to include this resources "release-service-configurator-role.yaml" in the kustomization.yaml file, the one in the same folder (component/release/kustomization.yaml)

Reply:
fixed

Reply:
looks better, I see the role in the output of kustomize build now